### PR TITLE
ENG-773 SEO fields for pages

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **jeffgo10**

If they are not already assigned, you can assign the PR to them by writing `/assign @jeffgo10` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/locales/it.js [hunk 1]
```diff
@@ -458,6 +459,8 @@ export default {
     'pages.pageForm.mimeType': 'MimeType',
     'pages.pageForm.mimeTypeHelp': 'Identifica il tipo di informazione (immagine, testo...) che Entando deve restituire al browser che le richiede',
     'pages.pageForm.saveAndConfigure': 'Salva e Configura',
+    'pages.pageForm.seoFriendlyCode': 'Codice Amichevole',
```
Comment (gidesan) [inline src/locales/it.js:462]:
```suggestion
    'pages.pageForm.seoFriendlyCode': 'SEO friendly code (slug)',
```

File: src/locales/it.js [hunk 2]
```diff
@@ -763,6 +766,17 @@ export default {
     'category.reference.table.description': 'Nome',
     'category.reference.table.typeCode': 'Tipo',
     'category.reference.table.lastModified': 'Ultima Modifica',
+    'app.pages.title': 'Titolo',
+    'app.seo.description': 'Descrizione di SEO',
```
Comment (gidesan) [inline src/locales/it.js:770]:
```suggestion
    'app.seo.description': 'Descrizione SEO',
```

File: src/locales/it.js [hunk 3]
```diff
@@ -763,6 +766,17 @@ export default {
...
+    'app.seo.keywords': 'Parole chiave di SEO',
```
Comment (gidesan) [inline src/locales/it.js:771]:
```suggestion
    'app.seo.keywords': 'Parole chiave SEO',
```

File: src/locales/it.js [hunk 4]
```diff
@@ -763,6 +766,17 @@ export default {
...
+    'app.seo.addMetatag': 'Aggiungi Metatag',
+    'app.seo.addMetatagKey': 'Chiave',
+    'app.seo.addMetatagValue': 'Valore',
+    'app.pages.titleHelp': 'Questo funge da titolo della tua pagina per scopi SEO',
```
Comment (gidesan) [inline src/locales/it.js:775]:
```suggestion
    'app.pages.titleHelp': 'Titolo della pagina per scopi SEO',
```

File: src/locales/it.js [hunk 5]
```diff
@@ -763,6 +766,17 @@ export default {
...
+    'app.seo.descriptionHelp': 'La meta descrizione è un frammento di testo che riassume il contenuto della pagina',
+    'app.seo.keywordsHelp': 'Le meta parole chiave sono un frammento di testo che indica le parole chiave principali della pagina, separate da virgola.',
+    'app.seo.addMetatagHelp': 'In questa sezione è possibile aggiungere un nuovo metatag (personalizzato o suggerito da un catalogo interno)',
+    'app.seo.inheritLangLabel': 'Eredita dalla lingua predefinita',
```
Comment (gidesan) [inline src/locales/it.js:779]:
```suggestion
    'app.seo.inheritLangLabel': 'Utilizza la lingua predefinita',
```

File: src/state/pages/actions.js [hunk 6]
```diff
@@ -394,8 +414,12 @@ export const sendPatchPage = pageData => async (dispatch, getState) => {
   }
 };
 
-export const fetchPageForm = pageCode => dispatch => fetchPage(pageCode)(dispatch)
+export const fetchPageForm = pageCode => dispatch => fetchPageInfo(pageCode)(dispatch)
   .then((response) => {
+    // const pagePayload = {
```
Comment (gidesan) [inline src/state/pages/actions.js]: can we remove this commented code?

File: src/ui/pages/common/SeoInfo.js [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+import { Tabs, Tab, Col, ControlLabel } from 'patternfly-react';
+import { Field, FieldArray } from 'redux-form';
+import { required, maxLength } from '@entando/utils';
+import RenderTextInput from 'ui/common/form/RenderTextInput';
+import FormLabel from 'ui/common/form/FormLabel';
+import SwitchRenderer from 'ui/common/form/SwitchRenderer';
+
+import SeoInfoMetadataContainer from 'ui/pages/common/SeoInfoMetadataContainer';
+
+// const msgTitle = defineMessages({
```
Comment (gidesan) [inline src/ui/pages/common/SeoInfo.js:12]: commented code

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/802#pullrequestreview-457537806" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**